.env

```diff
@@ -1,6 +1,5 @@
 REACT_APP_PWA=inactive
 REACT_APP_CARTARO_URL=//cartaro2.stag.trafimage.ch/api/v1/
-REACT_APP_GEOSERVER_URL=//maps.trafimage.ch/geoserver/trafimage/ows
+REACT_APP_BASE_URL=//maps.trafimage.ch
```

Comment:
I will  keep REACT_APP_GEOSERVER_URL and REACT_APP_DEPARTURE_URL . because you could want to test a different version for each  service .
use a REACT_APP_RESSOURCE_URL  to have an url where to get static content,.